src/library/scala/collection/immutable/BitSet.scala

```diff
@@ -16,7 +16,8 @@ import mutable.{Builder, GrowableBuilder}
   */
 @SerialVersionUID(3L)
 sealed abstract class BitSet
-  extends SortedSet[Int]
+  extends AbstractSet[Int]
+    with SortedSet[Int]
```

Comment:
Why not have `AbstractSortedSet`?

Reply:
It didn't seem worth it, as there are only two new methods in `SortedSet`